libs/core/src/lib/toolbar/toolbar-overflow-button-menu.directive.ts

```diff
@@ -0,0 +1,11 @@
+import { Directive } from '@angular/core';
+import { ToolbarItemDirective } from './toolbar-item.directive';
+
+@Directive({
+    // tslint:disable-next-line: directive-selector
+    selector: '[fd-toolbar-overflow-button-menu]',
```

Comment:
As I know we support two type of naming. camelCase and kebab-case. If you add camelCase option for the directive you no need to disable ts lint rule

Reply:
Modified.